custom_components/ramses_cc/climate.py

```diff
@@ -287,31 +293,40 @@ def svc_set_system_mode(self, mode, period=None, days=None) -> None:
         self._call_client_api(self._device.set_mode, system_mode=mode, until=until)
 
 
-class RamsesZone(RamsesZoneBase, ClimateEntity):
-    """Base for a Honeywell TCS Zone."""
+class RamsesZone(RamsesEntity, ClimateEntity):
+    """Representation of a Ramses zone."""
+
+    _device: Zone
 
     _attr_icon: str = "mdi:radiator"
     _attr_hvac_modes: list[str] = list(MODE_TO_ZONE)
+    _attr_precision: PRECISION_TENTHS
```

Comment:
The precision is 0.01, not 0.1.

People like this greater precision, and will complain if it's removed.

I would leave this 0.01 for now, then we'll make it optional via **configuration.yaml**.

Reply:
I copied this [as-is from RamsesZoneBase](https://github.com/zxdavb/ramses_cc/blob/3aa5f06ab9d0e759918801297c62753f6b5759be/custom_components/ramses_cc/__init__.py#L280).

Reply:
My understanding was that the precision increase was from 0.5 as is "normally" available to 0.1 (ie. PRECISION_TENTHS rather than PRECISION_HALVES).

Reply:
HA never allowed 0.01 for climate, water_heater... SO OK for that. But aim to keep for sensors.

We have a separate branch now - we can be more adventurous.